eth2prysm/values.yaml

```diff
@@ -0,0 +1,30 @@
+# Default values for eth2prysm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: prysm
+ethereumNetwork: pyrmont
+imageVersion: v1.3.2
+
+beacon:
+  name: beacon
+  label: beacon
```

Comment:
maybe just name is enough? if we really want to do `label`, then we should let them config a Map. For now we can just use `beaon.name` to config the label `app: name`.

Reply:
Changed them to use name instead.